astrokat/correlator.py

```diff
@@ -0,0 +1,26 @@
+# Setting correlator configuration values different from default values
+
+def set_fengines(session,
+        requant_gains=None,
+        fft_shift=None):
+    if not session.cbf.fengine.inputs:
+        msg = 'Failed to get correlator input labels, cannot set the F-engine gains'
+        raise RuntimeError(msg)
+
+    for inp in session.cbf.fengine.inputs:
+        # Set the gain to a single non complex number if needed
+        if requant_gains is not None:
+            # TODO: read and store values before assignment
+            session.cbf.fengine.req.gain(inp, requant_gains)
+            msg = 'F-engine {} gain set to {}'.format(
+                    str(inp), requant_gains)
+            user_logger.info(msg)
```

Comment:
You'll need to pass this function the logger reference.

Reply:
This is only a function stub and not implemented.
After the merge I will branch of for the correlator setting implementation needed for beamformer.

I would like to leave it in a state that will cause and error, if that is okay